pandas/plotting/_matplotlib/converter.py

```diff
@@ -152,7 +152,7 @@ def _to_ordinalf(tm: pydt.time) -> float:
 
 def time2num(d):
     if isinstance(d, str):
-        parsed = tools.to_datetime(d)
+        parsed = Timestamp(d)
```

Comment:
This probably changes error messages? If yes, should add a whatsnew

Reply:
i think this also makes the check/raise on the next two lines unnecesary

Reply:
yeah good call, looks like they're uncovered as it is anyway https://app.codecov.io/gh/pandas-dev/pandas/blob/main/pandas/plotting/_matplotlib/converter.py